.github/workflows/test.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           ghcup install ghc ${{matrix.ghc}}
           ghcup set ghc ${{matrix.ghc}}
 
-          ghcup install hls 1.4.0
+          ghcup install hls 2.2.0.0
```

Comment:
Actually, this is wrong because the test is supposed to test our ability to *fallback* to older HLS versions for unsupported GHC versions.

Reply:
But since the next release drops support for 8.10.7, this will be correct again :)

Reply:
Yes, thinking that we may not merge big changes before the next hls release, it's not bad to write it here now.